_posts/2016/2016-02-17-development-from-the-treches.md

```diff
@@ -0,0 +1,59 @@
+
+### Events
+In regular basis we organise after hours activities like lunches, dinners or sailing days!!!
+the company organises other events such as the ePagesYou, which is the convention of the company and takes place in Hamburg, and the Hackathon, when all developers meet for 2 days to create apps for our appstore.
+
+## Team
+Our team consists of 4 engineers, we are looking for a [QA](http://www.epages.com/en/career/jobs/QA_BCN-HH.php) and two more [developers](http://www.epages.com/en/career/jobs/software-developer.php). The QA, Scrum Master and Product Owner are located in Hamburg and support us in our daily basis work.
+Our team is responsible, together with another team in Hamburg, for the connectors to the e-commerce platform, such as payments, shippings and others.
+
+Also, as in the company we try to do our best to be as **Agile** as possible, we follow the **Scrum** principles, we do **pair programming**, we share our knowledge in daily code reviews, we do presentations, etc.
+
+### How do we organise ourselves?
+Maybe you are wondering how do we manage to organise ourselves without a scrum master and still have time to take a coffee?
+One of us plays the role of **scrum master** as well as developer, but when things get crazy we have the Hamburg scrum master to helps us.
+
+{% image dev-bc-group2.jpg 50% left %}{% endimage %}
+The **product owner** is also placed in Hamburg but we are in touch with him almost every day because he attends our dailies, reviews and plannings. He also comes to Barcelona once a month to discuss the progress of the different projects or plan the new ones.
+
+A sprint takes two weeks, after that we do the **review** inviting all colleagues, some are joining by video-conference and others personally. After that we take our time to do the **retrospective** with our backup scrum master and the **planning** for the next two weeks with the product owner. After the planning we sit together in order to split, analyse and assign tasks for the sprint.
+
+Every morning we do our daily stand-ups and after that a joint review of the code, which we have been doing since the last review. When coding we try to do pair programming and in early stages of a new development, we sit all together to model the new connector, draw **UML** diagrams or even code (we use a projector here).
+
+### Communication
+{% image video-connection.jpg 40% right %}{% endimage %}
+We use tools that let us stay in touch with the other teams in Germany, specially with the other 'connector' team. We have a permanent video-conference connection with our twin team. Also we have the **Community Of Practice** every second week where we share knowledge, issues and new ideas with other developers.
+
+To communicate with the whole firm we use **Hipchat**, which is a nice chat client, **Confluence** as Intranet to share knowledge, documentation, etc. and **Jira** to track our tasks and sprints.
+
+And that's all more or less, hope you like it!
```

Comment:
And that's all more or less, hope you like it! > And that's all more or less. I hope you enjoyed reading.